Tests/ApolloTests/AutomaticPersistedQueriesTests.swift

```diff
@@ -145,9 +145,6 @@ class AutomaticPersistedQueriesTests: XCTestCase {
       expect(file: file, line: line, version).to(equal(1))
 
       expect(file: file, line: line, sha256Hash).to(equal(O.operationIdentifier))
-
-    } else {
-      expect(file: file, line: line, ext).to(beNil())
```

Comment:
We're needlessly testing that `extensions` is empty; we shouldn't be enforcing that outside of what this test is concerned with, which is the persisted query keys in `extensions`.